### ISSUE TITLE
[P2][Sourcing] Freeze/Unfreeze popover for freezable columns are not accessible using keyboards.

### ISSUE THREAD

**Issue by OneEyedOwl123** [SAP/fundamental-ngx#12699]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Freeze/Unfreeze popups which open when we click/hit enter on freezable column header for fdp-tables should be acessible using keyboards.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular 15, version: 0.43.47

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
This issue is in fdp-tables with freezable columns.
1. Navigate to any freezable column using keyboard.
2. Hit "enter". It should open the freeze/unfreeze popover.
3. Try to freeze or unfreeze the column using keyboard only by hitting enter/spacebar again. 
Nothing happens. It is not accessible using keyboards.

https://github.com/user-attachments/assets/1fe45248-dba0-47b1-826e-075190f7e1d9

#### Please provide stackblitz example(s).
https://stackblitz.com/edit/angular-xwszvn?file=package.json

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
All interactable elements should be accessible by keyboards.

#### Did you check the documentation and the API?
yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
